views/js/runner/plugins/navigation/validateResponses.js

```diff
@@ -56,11 +56,18 @@ define([
          * @returns {this}
          */
         init: function init() {
-            this.getTestRunner().before('move', function () {
+
+            var pluginConfig = this.getConfig();
+
+            this.getTestRunner().before('move', function (e, direction) {
                 var self = this;
                 var testContext = this.getTestContext();
                 var isInteracting = !this.getItemState(testContext.itemIdentifier, 'disabled');
 
+                if (!pluginConfig.validateOnPreviousMove && direction === 'previous') {
+                    return Promise.resolve();
```

Comment:
sorry, I don't understand this part. This method returned Promise before, but now it can return the result of the resolve method, correct?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve
"The static `Promise.resolve` function returns a Promise that is resolved."